src/DataStructures.jl

```diff
@@ -52,7 +59,11 @@ module DataStructures
 
     import Base: eachindex, keytype, valtype
 
-    _include_string(str) = eval(parse(str))
+    @static if VERSION < v"0.7.0-DEV.2437"
+        _include_string(str) = eval(parse(str))
+    else
+        _include_string(str) = eval(Meta.parse(str))
```

Comment:
I think this can be used unconditionally, because `Meta.parse` is available on 0.6, too.

Reply:
Ah, right.